sdk/search/search-documents/test/public/utils/setup.ts

```diff
@@ -12,8 +12,9 @@ import {
 import { Hotel } from "./interfaces";
 import { delay } from "../../../src/serviceUtils";
```

Comment:
There is a internal reference here. Can you update this to ../../../src instead?
If the "delay" is not exported, then you may have to move the tests related to internal folder.
since the min-max script finds an internal reference here, it replaces it with empty file and that's why it gives build errors.
@HarshaNalluru

Reply:
Not doing it here to keep the changes minimal